schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json

```diff
@@ -33,22 +33,41 @@
               "type": "object",
               "additionalProperties": {
                 "type": "string"
-              }
+              },
+              "properties": {}
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
           "description": "Resource tags"
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/IdentityProperties"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The properties associated with the identity."
         }
       },
       "required": [
         "name",
         "type",
         "apiVersion",
-        "location"
+        "properties"
```

Comment:
`properties` is not required

Reply:
As below, I *think* marking properties as "readOnly" should solve this.